packages/react/src/components/header/Header.stories.tsx

```diff
@@ -131,6 +135,8 @@ export const WithFullFeatures = (args) => (
             <Header.Link
               href="#"
               label="Senioripalvelut"
+              openDropdownAriaButtonLabel="Avaa Senioripalvelut alavalikko"
```

Comment:
Just for future reference: Couldn't we just have ariaButtonLabel and some prefixes for open & close.